nominal/nominal.py

```diff
@@ -339,3 +348,41 @@ def _get_start_end_timestamp_csv_file(
         IntegralNanosecondsUTC(start.to_datetime64().astype(int)),
         IntegralNanosecondsUTC(end.to_datetime64().astype(int)),
     )
+
+def create_checklist(
+    assignee_email: str,
+    title: str,
+    checks: Sequence[CreateCheck],
+    checklist_variables: Sequence[CreateChecklistVariable],
+    default_ref_name: str | None = None,
+    commit_message: str | None = None,
+    description: str | None = None,
+    properties: Mapping[str, str] | None = None,
+    labels: Sequence[str] = (),
+    is_published: bool = True,
```

Comment:
what is is_published?

Reply:
In our backend checklist version control, you can save a workbook as a draft, or publish it, analogous to committing and merging to main (I think).

Reply:
got it so you could potentially `get_checklist()` which might return a draft?

Reply:
potentially we should defer touching drafts at all, and filter out drafts--unless folks think otherwise

Reply:
I think that's a good idea. I'm going to just throw if they try to get a checklist that isn't published